Modules/Forum/classes/class.ilObjForum.php

```diff
@@ -277,40 +268,6 @@ public function updateLastAccess(int $a_usr_id, int $a_thread_id): void
         }
     }
 
-    public static function _updateOldAccess(int $a_usr_id): void
```

Comment:
Are you sure that we can remove the complete method? If yes, we should also remove the database table in a new database update step (maybe here Modules/Forum/classes/Setup/class.ilForumDatabaseUpdateSteps.php or in a new class).